proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -51,66 +71,82 @@ definition pspace_no_overlap' :: "obj_ref \<Rightarrow> nat \<Rightarrow> kernel
      \<lambda>s. \<forall>x ko. ksPSpace s x = Some ko \<longrightarrow>
                 (mask_range x (objBitsKO ko)) \<inter> {ptr .. (ptr && ~~ mask bits) + mask bits} = {}"
 
-definition ko_wp_at' :: "(kernel_object \<Rightarrow> bool) \<Rightarrow> obj_ref \<Rightarrow> kernel_state \<Rightarrow> bool" where
+definition
```

Comment:
This might have been part of the automatic patch, but we shouldn't remove the type signature here. I've explicitly added a lot of these in the Invariants file to improve style and maintainability. Would be sad to lose all that in the merge.